src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1878,7 +1883,7 @@
         "kinds": {
           "description": "A set of distinct strings that categorize the relationshship. Well-known kinds include canPrecede, canFollow, willPrecede, willFollow, superset, subset, equal, disjoint, relevant, and incomparable.",
           "type": "array",
-          "minItems": 1,
+          "default": [ "relevant" ],
```

Comment:
>default": [ "relevant" ], [](start = 11, length = 25)

this is currently breaking the validator. i will take a look at it tomorrow with a fresh mind. Raising this PR as a draft for awareness.